<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wCreateDB</class>
 <widget class="QWidget" name="wCreateDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成DB</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnSelectSqlFile">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnSelectCreateDB">
        <property name="text">
         <string>选择数据库</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="lbSplitChar">
        <property name="text">
         <string>分隔符</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QToolButton" name="btnSave">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>34</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="btnClearCreateDB">
        <property name="toolTip">
         <string>清空最新打开的数据库文件列表</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="btnClearSqlFile">
        <property name="toolTip">
         <string>清空最新打开的SQL文件列表</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnExec">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>34</height>
         </size>
        </property>
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btnRefresh">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>34</height>
         </size>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLineEdit" name="edSplitChar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>$</string>
        </property>
        <property name="cursorPosition">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbCreateDB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbSqlFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbCreateDb">
        <property name="text">
         <string>数据库：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbSqlFile">
        <property name="text">
         <string>文件名：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="edSql"/>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="pbExecSql">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
